### PR TITLE
docs: correct comment on data

### LINKED ISSUE #209: README incorrectly documents data in options
#### Steps to reproduce

  1. Learn english.
  2. Read docs.
  3. Realise that the documentation for the data property of the options parameter is incorrect as the data is not base64 encoded.

### PR DESCRIPTION
**Pull request by strideynet**
The previous comment is incorrect, as data is not base64 encoded.

Fixes #209 🦕

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ gaxios.request({url: '/data'}).then(...);
   // The HTTP methods to be sent with the request.
   headers: { 'some': 'header' },
 
-  // The data to base64 encode and send in the body of the request.
+  // The data send in the body of the request. Objects will be converted to JSON.
```
Comment (bcoe) [inline README.md]: perhaps let's just say:

```
The data to send in the body of the request.
```

Comment (strideynet) [reply README.md]: Hmm. I was trying to capture the underlying behaviour in a way that would make sense to the end developer. It seems at the moment there is some special handling for streams and that Javascript objects are converted to JSON. 

Although perhaps you are right that simpler is better. I have so far found that whilst this library is pretty good, the documentation tends to not always match the behaviour so I'd rather it was correct and vague than incorrect and precise.

Comment (bcoe) [reply README.md]: @strideynet I see your point, perhaps just fix the typo `The data send` to `The data to send`, and perhaps:

```
Data objects will be serialized as JSON
```

:point_up: streams are technically an object as well, so something that calls out the fact that this refers to the behavior of [POJOs](https://en.wikipedia.org/wiki/Plain_old_Java_object) (I think perhaps more folks will understand "data object" than POJO).

Comment (strideynet) [reply README.md]: That's fair enough. I'll incorporate that change.

**Comment by bcoe** [comment]
@strideynet thank you for taking the time to make a contribution 👌

**Comment by strideynet** [comment]
@bcoe It's a pleasure. I've noticed a few areas in Gaxios that could do with improving so I may well be submitting issues/patches in the next few weeks :)

**Comment by bcoe** [comment]
@strideynet :+1: always happy to have community contributions.

**Pull request merged**